### ISSUE TITLE
Questions about the pre-trained model

### ISSUE THREAD

**Issue by Xingxl2studious** [alexandra-chron/relm_unmt#1]
Hi!

First off, thank you in advance for any advice you can offer.

I am also working on low-resource NMT. I wonder whether you can share with me the pre-trained English monolingual masked LM(mono_mlm_en_68m.pth). That will help my research a lot.

I apologize if the answer to this question is obvious!

Many thanks. :)

**Comment by alexandra-chron** [comment]
Hey,

Sure, I don't think there's any problem with releasing the pretrained model! 

Here's the link to the checkpoint https://www.dropbox.com/s/nrhnyyjuncbwx71/pretrained_en_mlm.zip?dl=0 -- the vocabulary and BPE codes are also there, to make preprocessing of new data easier. I also uploaded the validation and test set. 

Good luck with your experiments!

**Comment by Xingxl2studious** [comment]
Thanks a lot for sharing that. : )
